crates/winmd/src/types/delegate.rs

```diff
@@ -60,7 +60,7 @@ impl Delegate {
 
         quote! {
             #[repr(transparent)]
-            #[derive(Default)]
+            #[derive(Default, PartialEq)]
```

Comment:
Derives do the conservative thing and place a matching constraint on all generics for the derived trait. So for example:
```rust
#[derive(PartialEq)]
struct Foo<T> {
  f: std::marker::PhantomData<T>
}
// Foo<T> is only PartialEq when T: PartialEq
```
Is this what we want? Or do we just want to differ to the `self.ptr == other.ptr`? If so, we need to implement `PartialEq` ourselves.

Reply:
Ah right. Yes, we just want to compare the pointers.

Reply:
https://github.com/microsoft/winrt-rs/issues/140